### DISCUSSION TITLE
Resize an image in an ojs chuck (presentation revealjs)

### DISCUSSION THREAD

**Discussion by neocarto** [quarto-dev/quarto-cli#3237] [Q&A]
Is there a way to resize an image in an ojs chuck ? here for example, how to reduce the size of this map so that it can fit in a reveal.js presentation ? And how to change the style of the slider (because the text is big) ?

```{ojs}
viewof k = Inputs.range([-100, 100], {
  step: 10,
  value: 0
})
import { map } with { k as k } from "@neocartocnrs/how-to-lie-with-maps"
map
```

![Screenshot from 2022-11-08 11-48-56](https://user-images.githubusercontent.com/8111373/200545386-494fbbfc-0326-4a07-acf8-dc8d0f3be3bd.png)

**Comment by cscheid** [comment]
> here for example, how to reduce the size of this map so that it can fit in a reveal.js presentation ? 

I can't seem to reproduce this issue. When I resize the window, that imported map resizes dynamically for me:


https://user-images.githubusercontent.com/285675/200611629-698b7616-3c35-4e67-8833-40f1ed457aff.mov



> And how to change the style of the slider (because the text is big) ?

This is something we'd like to do better, but there's no out-of-the-box support right now. Still, you should be able to use CSS to do it.

**Reply by neocarto** [reply]
Yes, the map is resized when I minimize the window. But by default, the map goes over in height. So I was wondering if it was possible to fix  the size of a figure, in the ojs chuck settings,

**Reply by cscheid** [reply]
We can't do that in general because the standard flow of both observablehq documents _and_ html documents is vertical. If we use automatic _height_ resizing by default, then you'd have every figure taking all of the screen. You will need to create your own style for it. Adding something like `max-height: 50vh` or something like that should work (I tried it). Note that your SVG will need fixing, because it doesn't clip the map horizontally:

<img width="1484" alt="image" src="https://user-images.githubusercontent.com/285675/200868938-3b501cbb-65ea-49f8-a5b0-93dafeccf91d.png">
